Makefile

```diff
@@ -269,7 +273,6 @@ deploy: manifests kustomize ## Deploy controller and agent to the K8s cluster sp
 	  $(SED) -e 's@gkm\.agent\.image=.*@gkm.agent.image=$(AGENT_IMG)@' \
 	      -e 's@gkm\.csi\.image=.*@gkm.csi.image=$(CSI_IMG)@' \
 		  kustomization.yaml.env > kustomization.yaml
-	$(KUSTOMIZE) build config/default | $(KUBECTL) apply -f -
```

Comment:
Where does this get done?

Reply:
`config/kind-gpu` is an overlay of `config/default`,  so `$(KUSTOMIZE) build config/default | $(KUBECTL) apply -f -` is done when we Kustomize build `config/kind-gpu`